Code/Engine/Foundation/Memory/AllocatorBase.h

```diff
@@ -39,15 +39,15 @@ class EZ_FOUNDATION_DLL ezAllocatorBase
 
   /// \brief Interface, do not use this directly, always use the new/delete macros below
   virtual void* Allocate(size_t uiSize, size_t uiAlign, ezMemoryUtils::DestructorFunction destructorFunc = nullptr) = 0;
-  virtual void Deallocate(void* ptr) = 0;
-  virtual void* Reallocate(void* ptr, size_t uiCurrentSize, size_t uiNewSize, size_t uiAlign);
+  virtual void Deallocate(void* pPtr) = 0;
```

Comment:
Should we allow `ptr` as special name similar to `lhs` ?

Reply:
I wouldn't add too many exceptions. These are special cases and it could also be called "pMemory" or so instead.